css-src/sass/_header.scss

```diff
@@ -0,0 +1,56 @@
+// Stylesheet also used in tdr-transfer-frontend repo: https://github.com/nationalarchives/tdr-transfer-frontend
```

Comment:
Could you add a link back to this file in https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/npm/css-src/sass/_header.scss#L1?

Reply:
There's a PR for adding the comment in https://github.com/nationalarchives/tdr-transfer-frontend/pull/758